### PR TITLE
fix(docs): removing of hardcoded density

### PR DESCRIPTION
**Pull request by platon-rov**
## Related Issue(s)

Refers to https://github.com/SAP/fundamental-ngx/issues/8982

## Description

Removed hard-coded content density from some examples where it's not supposed.

### CODE REVIEW

File: libs/docs/core/calendar/examples/calendar-mobile-example/calendar-mobile-example.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
             <fd-button-bar
                 fdType="emphasized"
                 label="Ok"
-                [fdCompact]="false"
+                fdCozy
```
Comment (InnaAtanasova) [inline libs/docs/core/calendar/examples/calendar-mobile-example/calendar-mobile-example.component.html:17]: isn't this also "hardcoding" the density?

**Review by InnaAtanasova — COMMENTED** [review]
Just one comment for fdCozy, the rest looks great

**Review by g-cheishvili — APPROVED** [review]

**Comment by g-cheishvili** [comment]
e2e fail, because there are tests for compacts

**Review by mikerodonnell89 — COMMENTED** [review]
Should we be removing entirely any examples that solely exist to show the hardcoded density? It is odd that some have components have this and some do not

**Comment by droshev** [comment]
> Should we be removing entirely any examples that solely exist to show the hardcoded density? It is odd that some have components have this and some do not

I see @mikerodonnell89 's point. There is no point have separate examples for cozy and compact.

**Comment by platon-rov** [comment]
> I see @mikerodonnell89 's point. There is no point have separate examples for cozy and compact.

yea, we have ticket for it https://github.com/SAP/fundamental-ngx/issues/8982, you said that would be good task for new intern

**Pull request merged**